neuralop/models/integral_transform.py

```diff
@@ -6,7 +6,47 @@
 from .mlp import MLPLinear
 
 class IntegralTransform(nn.Module):
-    def __init__(self, mlp=None,
+    """Integral Kernel Transform (GNO)
+    Computes one of the following:
+        (a) \int_{A(x)} k(x, y) dy
```

Comment:
here dy corresponds to the actual measure (ie this is always the weighted integral)?